Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh

```diff
@@ -157,16 +157,14 @@ case $DISTRO in
         if [ 0 -eq $exit_code ]; then
             LogMsg "Exiting with state: $__LIS_TESTCOMPLETED."
             SetTestStateCompleted
-            exit 0
         else
             LogMsg "Exiting with state: $__LIS_TESTABORTED."
             SetTestStateAborted
-            exit 1
```

Comment:
I am confused. SetTestStateAborted leads 'exit 1' in many places, but some lines have 'exit 0'. I remember 'Abort' derived the negative situation. Should we stick with 'exit 1'? @LiliDeng @johnsongeorge-w comment?

Reply:
In this commit I tried removing all ```exit 1``` to avoid the retries from framework. Current pointed code is removing exit 1.

Reply:
SetTestStateAborted this will decide test result, exit 1 or 0 doesn't, but if we didn't set ignoreexitcode in powershell side, and exit with a non-zero, then the script will be executed again and again.

Reply:
I think we should keep 'exit' statement with SetTestStateXXX call. We should have the document update.

Reply:
No, we should exit with 0 after setting up the appropriate state.

Reply:
I meant 'exit 1' not 'exit 0'

Reply:
No exit required here.